Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderIndex.spec.json

```
@@ -82,10 +82,18 @@
                     "path": {
                         "type": "string"
                     },
-                    "isExternal": {
+                    "deprecated": {
                         "type": "boolean",
                         "default": "false"
                     },
+                    "external": {
+                        "type": "boolean",
+                        "default": "false"
+                    },
+                    "beta": {
+                        "type": "boolean",
+                        "default": "false"
+                    }
```

Looks like we're missing a comma here.

```suggestion
                    },
```

We don't (yet) have CI for validating our OpenAPI schemas so I always find it helpful to manually run them through https://apitools.dev/swagger-parser/online/ whenever making a change.

---

Thanks for the heads up! Maybe we should run this as part `./bin/test`?